.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: Release
+
+on:
+  push:
+    branches: ["master"]
+
+jobs:
+  release:
```

Comment:
I suggest adding an environment to this job

Reply:
Also, a check to see if the repo is upstream

Reply:
> Do you mean like environment: release below line 10 ? I like that idea!
yes, exactly

about the check for upstream: run the job only if it's running of the upstream repository `jupyter-incubator/sparkmagic` not a fork `ljubon/sparkmagic`